components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -58,19 +58,21 @@
         private String protocol;
         private String hostName;
         private int port;
+        private String proxyContextPath;
         private String urlPath;
         private Map<String, String> parameters;
         private String fragment;
         private String absoluteUrl;
         private String relativeUrl;
 
-        private ServiceURLImpl(String protocol, String hostName, int port, String urlPath,
+        private ServiceURLImpl(String protocol, String hostName, int port, String proxyContextPath, String urlPath,
```

Comment:
proxyContext or proxyContextPath?

Reply:
Should it be renamed to proxyContext? Used "proxyContextPath" since it was used here https://github.com/wso2/carbon-identity-framework/blob/4f6342ef09b98348a076679ea6921b3af02019d0/components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java#L33

Reply:
Okay let's go with that